DOCS/canonical-identity-platform-tutorial-deploy-identity-bundle.md

```diff
@@ -1,47 +1,25 @@
-# Introduction
+The Canonical Identity Platform is a composable identity broker based on open source products from [Ory](https://www.ory.sh/open-source/), [Postgres](https://www.postgresql.org/) and [Traefik Labs](https://traefik.io/).
 
-The Canonical Identity Platform is a composable identity broker and identity provider based on open source products from [Ory](https://www.ory.sh/open-source/), [Postgres](https://www.postgresql.org/) and [Traefik Labs](https://traefik.io/).
+> See more: [Charmhub | Identity Plaform](https://charmhub.io/identity-platform)
 
-The core components of the Canonical Identity Platform are grouped in a [Juju Bundle](https://juju.is/docs/juju/bundle). In this tutorial you will:
-
-1. Deploy the Canonical Identity platform bundle
-2. Register a client application in Azure AD
-3. Configure Azure AD as an external identity provider
-4. Deploy Charmed Grafana and add single sign on
+The core components of the Canonical Identity Platform are grouped in a Juju Bundle. In this tutorial you will deploy the Identity Platform juju Bundle and use it to provide SSO to an OIDC compatible Charmed application.
```

Comment:
AFAIK charm refers to the sidecar that manages the workload container or to the set of workload container + sidecar. Here we care about an application that is managed by a charm (charmed app).

Am I wrong?

Reply:
> AFAIK charm refers to the sidecar that manages the workload container or to the set of workload container + sidecar. Here we care about an application that is managed by a charm (charmed app).
> 
> Am I wrong?

You are not wrong. It's our terminology that has an issue, because we use "charm" for a lot of different things; see, e.g., https://discourse.charmhub.io/t/what-do-we-mean-when-we-say-charm/8603/11 . That said, the meaning where charm = charmed application is very strong, and all our high-level terminology is aligned with it. (E.g., with Juju you manage charms, the SDK is called the "charm SDK", the file you obtain when you charm an application has the extension `.charm`, our store is called _Charm_hub, etc.)